adapters/postgres/queries.go

```diff
@@ -45,7 +46,7 @@ func (adapter *Postgres) GetScript(verb, folder, scriptName string) (script stri
 
 // ParseScript use values sent by users and add on script
 func (adapter *Postgres) ParseScript(scriptPath string, templateData map[string]interface{}) (sqlQuery string, values []interface{}, err error) {
-	_, tplName := path.Split(scriptPath)
+	_, tplName := path.Split(strings.ReplaceAll(scriptPath, "\\", "/"))
```

Comment:
https://pkg.go.dev/path/filepath#Split replace is not required if use split from filepath pkg

Reply:
The previous code failed when actually running on amd64, windows 10, and go1.16.6. I revised it based on actual running and debugging, and it worked normally after the revision.

Reply:
I have changed the method you suggested, although the effect is the same, obviously your method is more suitable for this scenario. Thanks @felipeweb!